File: python/예산.py
# def solution(d, budget):
#     tmp = 0
#     answer = 0
#     d.sort()
    
#     for i in range(len(d)):
#         tmp += d[i]
#         if tmp > budget:
#             break
#         else:
#             answer += 1
    
#     return answer

def solution(d, budget):
    d.sort()
    while budget < sum(d):
        d.pop()
    return len(d)

File: js/예산.js
function solution(d, budget) {
    let tmp = 0;
    var answer = 0;
    let arr = d.sort((a,b) => a-b);

    for (let i=0; i < arr.length; i++) {
        tmp += arr[i]
        if (tmp > budget) {
            return answer
        } else {
            answer += 1
        }
    }
    return answer;
}